### PR TITLE
Fix documentation spelling and grammatical errors

### PR DESCRIPTION
**Pull request by bmorelli25**
Cleaned up some spelling and grammar issues I found in the docs. No content changes, so probably only needs to be reviewed by one of you?

### CODE REVIEW

File: docs/custom-spans.asciidoc [hunk 1]
```diff
@@ -7,14 +7,14 @@ endif::[]
...
-
 This is an example of how to use custom spans.
 For general information about the Elastic APM Node.js Span API,
 see the <<span-api,Span API documentation>>.
 
-In the example below we create an Express app that times how long it takes to:
+If you want to track and time a custom event that happens in your application during a transaction,
+you can add a new span to an existing transaction.
+
+In the example below, we create an Express app that times how long it takes to:
```
Comment (Qard) [inline docs/custom-spans.asciidoc]: Any particular reason for rearranging these paragraphs?

File: docs/express.asciidoc [hunk 2]
```diff
@@ -9,8 +9,7 @@ endif::[]
 
 Getting Elastic APM set up for your Express app is easy,
 and there are various ways you can tweak it to fit your needs.
-
-Follow the guide below and refer to the <<api,API documentation>> for all the advanced stuff.
+Follow the guide below to get started, then view the <<api,API Reference>> for the more advanced stuff.
```
Comment (Qard) [inline docs/express.asciidoc]: Maybe we should avoid the word "stuff" while you're in here? 😅

File: docs/hapi.asciidoc [hunk 3]
```diff
@@ -9,8 +9,7 @@ endif::[]
 
 Getting Elastic APM set up for your hapi app is easy,
 and there are various ways you can tweak it to fit your needs.
-
-Follow the guide below and refer to the <<api,API documentation>> for all the advanced stuff.
+Follow the guide below to get started, then view the <<api,API Reference>> for the more advanced stuff.
```
Comment (Qard) [inline docs/hapi.asciidoc]: Same comment about "stuff"

File: docs/lambda.asciidoc [hunk 4]
```diff
@@ -9,8 +9,7 @@ endif::[]
 
 Getting Elastic APM set up for your lambda functions is easy,
 and there are various ways you can tweak it to fit your needs.
-
-Follow the guide below and refer to the <<api,API documentation>> for all the advanced stuff.
+Follow the guide below to get started, then view the <<api,API Reference>> for the more advanced stuff.
```
Comment (Qard) [inline docs/lambda.asciidoc]: Again with the "stuff"

File: docs/restify.asciidoc [hunk 5]
```diff
@@ -9,8 +9,7 @@ endif::[]
 
 Getting Elastic APM set up for your Restify app is easy,
 and there are various ways you can tweak it to fit your needs.
-
-Follow the guide below and refer to the <<api,API documentation>> for all the advanced stuff.
+Follow the guide below to get started, then view the <<api,API Reference>> for the more advanced stuff.
```
Comment (Qard) [inline docs/restify.asciidoc]: Yet more "stuff"

File: docs/express.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/express.asciidoc]: 😆I was trying to only fix glaring grammatical issues in this update and focus on content later, but I agree, I'll just get rid of it.

Comment (Qard) [reply docs/express.asciidoc]: I'm good merging this as-is. Just a comment.

File: docs/custom-spans.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/custom-spans.asciidoc]: I was trying to match the format used in `custom-transactions.asciidoc`. That page opens with a similar sentence, so it made sense to me to start both pages in the same manner. 
screen shot 2018-10-24 at 12 53 55 pm

File: docs/express.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/express.asciidoc]: Nah I agree, let's just fix it now. Updated the branch

**Comment by Qard** [comment]
By the way, the CI failure is because we require subsystem prefixes in our commit messages. If you update your commits to have the `docs` prefix it should work. eg: `docs: change some things`

**Comment by bmorelli25** [comment]
Ahhh, thanks @Qard! I amended my commit messages to include `docs` prefix. 👨‍🎓

**Comment by bmorelli25** [comment]
Looks like I can't use uppercase either. Will fix again later tonight

**Comment by bmorelli25** [comment]
Checks passed 🎉!
@Qard - look good?

**Comment by Qard** [comment]
Oh, another note: it helps to have your issue names and content match what you intend to be in the final squashed commit title and message.

**Pull request merged**